backend/src/models/analysis.py

```diff
@@ -106,3 +106,25 @@ def units_to_annotate(self):
                     })
 
         return units
+
+    def find_discussion_post(self, discussion_post_id):
+        """ Finds a specific discussion post in an analysis by the discussion post id """
+        for discussion in self.discussions:
+            if discussion['post_id'] == discussion_post_id:
+                return discussion
+
+        return None
+
+    def find_authored_discussion_post(self, discussion_post_id, client_id):
+        """ Finds a discussion post from a user that authored the post in an analysis """
+        discussion_post = self.find_discussion_post(discussion_post_id)
+
+        print(discussion_post)
```

Comment:
Can remove the print/debugging statement

Reply:
This has been removed. 🖨️